dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -259,8 +259,7 @@ def replace_hostname(url)
 end
 
 Then /^element "([^"]*)" does not exist/ do |selector|
-  instances = @browser.execute_script("return $('#{selector}')");
-  instances.should eq []
+  @browser.execute_script("return $('#{selector}').length").should eq 0
```

Comment:
Out of curiosity, what was IE returning for this step?  (Or for that matter, why were other browsers returning empty array?  Interesting cast.)